src/pages/Overview/NoBridgeFeesBanner.tsx

```diff
@@ -0,0 +1,39 @@
+import { BodyLg, Stack, Image, VStack } from "@threshold-network/components"
+import {
+  AnnouncementBannerContainer,
+  AnnouncementBannerContainerProps,
+} from "../../components/AnnouncementBanner"
+import ButtonLink from "../../components/ButtonLink"
+import santaIsComing from "../../static/images/santa-is-coming.svg"
+import noBridgeFeesText from "../../static/images/no-bridge-fees-text.svg"
+import { FC } from "react"
+
+export const NoBridgeFeesBanner: FC<AnnouncementBannerContainerProps> = ({
+  ...props
+}) => {
+  return (
+    <AnnouncementBannerContainer
+      hideCloseBtn
+      py={12}
+      backgroundImage={`linear-gradient(to right, rgba(125,0,255,0.5),rgba(125,0,255,0.1)), url('${santaIsComing}')`}
+      backgroundSize={"cover"}
```

Comment:
It turns out Figma interprets blending modes its own way - not exact the same as CSS implementations so it's impossible to achieve pixel perfect effect.
```suggestion
           backgroundImage={`url(${santaIsComing}),linear-gradient(8deg, hsla(269, 100%, 50%, 100%) 20%, hsla(269, 100%, 50%, 35%) 70%)`}
      backgroundBlendMode={"color-dodge"}
      filter={"contrast(1.05) saturate(1.05) brightness(1.02)"}
```
With help of CSS filters I was able to finetune the effect. Not pixel perfect but quite decent.

Reply:
That's a great solution, but I've actually implemented Sorin solution from https://github.com/threshold-network/token-dashboard/pull/702#issuecomment-1863218510. I've exported the image with already applied overlay. See fbeda3dea950a4458212bad0bd5f1e5bbfd385d2

If you think that there are any cons of that solution please let me know.

Reply:
Yup, in fact it doesn't make sense to do it with code if we don't manipulate it (effects, animations etc). 
You proposed solution with CSS overlay and I wanted to keep it but the image with overlay already applied is more suitable. GJ 💪🏻